yuca/wrappers/coevolution_wrapper.py

```diff
@@ -0,0 +1,85 @@
+import numpy as np
```

Comment:
Can we get rid of this file?

Reply:
Yes, removed in [a53af5d](https://github.com/riveSunder/yuca/pull/50/commits/a53af5d4ebfa85b186d3c140e8f3f80be7364742) with no problems.